kaggle_environments/envs/halite/helpers.py

```diff
@@ -167,42 +41,17 @@ def player(self) -> int:
         """The current agent's player index."""
         return self["player"]
 
-    @property
-    def step(self) -> int:
-        """The current step index within the episode."""
-        return self["step"]
-
-    @property
-    def remaining_overage_time(self) -> int:
-        """The current step index within the episode."""
-        return self["remainingOverageTime"]
-
 
-class Configuration(ReadOnlyDict[str, any]):
+class Configuration(Configuration):
     """
     Configuration provides access to tunable parameters in the environment.
     This provides bindings for the configuration type described at https://github.com/Kaggle/kaggle-environments/blob/master/kaggle_environments/envs/halite/halite.json
     """
-    @property
```

Comment:
New base configuration and observation types mean we now don't have to redefine these fields for each environment.